actor/v7action/service_instance_details.go

```diff
@@ -97,6 +93,26 @@ func (actor Actor) GetServiceInstanceDetails(serviceInstanceName string, spaceGU
 	return serviceInstanceDetails, Warnings(warnings), nil
 }
 
+func (actor Actor) GetServiceInstanceParameters(serviceInstanceName string, spaceGUID string) (ServiceInstanceDetails, Warnings, error) {
```

Comment:
It feels like a `GetServiceInstanceParameters()` function should return `ServiceInstanceParameters` rather than `ServiceInstanceDetails`